examples/world3/figures/capital.jl

```diff
@@ -25,7 +25,7 @@ fig_3_36_variables = [
     (ag.aiph,  0, 50,    "aiph"),
 ]
 
-plotvariables(sol, (t, 1900, 1970), fig_3_36_variables, name="Fig. 3.36", showlegend=true, showaxis=true, colored=true)
+plotvariables(sol, (t, 1900, 1980), fig_3_36_variables, name="Fig. 3.36", showlegend=true, showaxis=true, colored=true)
```

Comment:
@universmile, Paulo says you fixed this but you still have to update this PR then.

Reply:
@natema, the PR has been updated.